server/utils.js

```diff
@@ -800,11 +800,15 @@ function removeServicePath(headers) {
   return fiwareHeaders;
 }
 
-function returnResponse(data, res) {
+function returnResponse(data, res, date) {
   res.statusCode = data.statusCode;
   res.headers = data.headers;
   if (data.body) {
-    res.json(dataToSchema(data.body));
+    if (date) {
```

Comment:
this is really strange in JS. if date is an optional parameter then it should be handled by the dataToSchema function and not using the code you propose